rakelib/kokoro/kokoro.rb

```diff
@@ -76,6 +76,22 @@ def cloudrad
     end
   end
 
+  def devsite
+    run_ci @gem do
+      header "Building Devsite Docs"
+      FileUtils.remove_dir "doc" if Dir.exists? "doc"
```

Comment:
Actually, one thing I just noticed. Please pass `true` as the second parameter to `FileUtils.remove_dir`, to prevent it from erroring if the directory is not present. (See https://ruby-doc.org/stdlib-3.0.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-remove_dir). Same in the `cloudrad` method.